### ISSUE TITLE
title poster_betterport

### ISSUE THREAD

**Issue by massonix** [brentthorne/posterdown#100]
Dear Brent,

I love the betterport design, I think it will look great in my next scientific conference. With that, I'd like to ask you if it would be possible to place the poster title and the authors in the green area of the poster (instead of the main findings) while keeping its current formatting.

Thanks a lot,

Ramon

**Comment by brentthorne** [comment]
Hello @massonix !

Glad to hear you enjoy posterdown :tada: 

Before I can make suggestions I would need to know what poster template you are thinking of using. I assume it would be either the `posterdown_betterland` or `posterdown_betterport`. 

As for adding the title into the main section I would ask if you are planning to also include the "main finding" or if the title will just replace it.

As for the author and institute information there would need to be a new template made which would change the associated `template.html` file. This isn't too challenging if you have experience with HTML and pandoc but if you don't it may require some info. 

Basically the two options once I get that info from you are either 1) for you to fork this repo > create a new branch > edit the template, or 2) You give me the info and I can do my best to make it work but can't guarantee that I'll be able to get to it on any sort of timeline unfortunately. 

Hopefully this helps, one day I plan to have a tutorial series up on my youtube channel to go over creating custom layouts but that has an unknown timeline as well :smile:
